lib/generators/alchemy/install/install_generator.rb

```diff
@@ -59,6 +59,10 @@ def copy_yml_files
         end
       end
 
+      def copy_config_yml
+        copy_file "config.yml", app_config_path.join("alchemy", "config.yml")
```

Comment:
maybe use a template and use ERB to add the values from to ruby config files?

Reply:
Done!